base_layer/wallet_ffi/tests/conftest.py

```
@@ -0,0 +1,135 @@
+    """Provide mock callback registry for testing."""
+    # Clear any previous state
+    mock_callback_registry.clear_stats()
+    yield mock_callback_registry
+    # Clean up after test
+    mock_callback_registry.clear_stats()
+
+
+@pytest.fixture
+def callback_tracker():
+    """Provide a fresh callback tracker for each test."""
+    from fixtures.mock_callbacks import MockCallbackRegistry
```

Flake8 (7.2.0)

[error] redefinition of unused 'MockCallbackRegistry' from line 10

(F811)